### DISCUSSION TITLE
Flopp.net no longer shows Google maps. Solution : a link to show on google ?

### DISCUSSION THREAD

**Discussion by olilesp** [2Abendsegler/GClh#2363] [Questions]
Hello.

As of January 2023, flopp.net has make amny changes to their website and maps, and they're no longer uses Google Maps, this had to be discontinued due to issues with Google's Terms of Service.
They are still using OpenStreetMaps, and now Stamen Land, Arcgis World Imagery.
Will the links be changed soon?

And is it possible to add a link to see a geocache with Google satellite, map, directly in the listing of the cache ?

[Flopp.net blog](https://blog.flopp.net/the-technology-behind-flopps-map/)

Thx from Belgium

**Comment by 2Abendsegler** [comment]
Hello,

I don't know exactly what you mean in your first question. The link to the Flopp's Map still works. What exactly should we change?

To your second question: 

There is already a link to Google Maps above the small map in the listing. On the Google Maps page you can then select the satellite map. The link is marked in blue in the image.

The script Geocaching Map Enhancements (GME) also shows Google Maps satellite in the small map in the listing. You can also start there with the satellite map. It then looks like in the image. You can also select other maps via the map selection, marked in red. Information about GME can be found [here](https://github.com/2Abendsegler/GME).

![Unbenannt](https://user-images.githubusercontent.com/22332216/219974942-3187164c-032c-4621-88d9-71787da0a4be.JPG)

Greetings from Germany
Frank

**Comment by olilesp** [comment]
Sorry, I may have misspoken, but I meant in the "show on Flopp's Map" pop up list as shown in the screenshot 1.
Flopp no ​​longer provides links for map preview with Google. It would therefore be necessary to remove the 4 Google links and perhaps replace them with the links of the other map layers offered by Flopp (Screenshot 2).

For my second question, I was thinking of asking for an addition of "Show on Google maps" with a pop up list including the various layers of Google maps, like "Show on Flopp's Map" and "Show on Openrouteservice" to show the geocache point on Gmaps.
It is true that a link exists further down the cache page. Sorry, I don't use that overlay option and link very much.


Screenshot 1 
![Capture d’écran 2023-02-20 114910](https://user-images.githubusercontent.com/35106228/220086579-03194617-21ac-4c17-ae9c-eb746b7abdcd.jpg)

Screenshot 2
![Capture d’écran 2023-02-20 120029](https://user-images.githubusercontent.com/35106228/220086962-b6438818-e78a-4dbb-9eb5-ad7467a26f28.jpg)

**Reply by 2Abendsegler** [reply]
To 1. Oh, I forgot about this pop up list for Flopp's Map. 😁 You are quite right, I opened an issue #2364 about it.

To 2. I also opened a feature request #2365. I don't know yet if and how we'll implement that, we'll see. I'll have to take a closer look at what can be done there.

**Comment by 2Abendsegler** [comment]
@olilesp

Hi,

to 2.
I tried to create the google maps popup for the listing. Unfortunately there doesn't seem to be a way to add additional waypoints like there is for example for the Flopp's Map links. So you just have the list coordinates which the map is then centered on.

In theory, we could also create route descriptions with intermediate points, like the other popups in the listing with routing functionality, for example at openrouteservice. But I really don't want that. It's also not what you had on the flops map before.

Actually you want to see the waypoints on the map like it is on the flops map. Just again with the map layers of Google Maps, right?

Please try the following. Please install GME as I mentioned before. And then please look at [this](https://www.geoclub.de/forum/t/geocaching-map-enhancements.82339/post-1349690) posting. It's in German but that's not so bad. Click on "Spoiler: Browse Map mit Wegpunkten eines Caches" and watch the short video. If you follow these instructions you then have the waypoints of a cache in the browse map and can use the Google Maps layer that we offer there. 
Maybe this helps?

Please let me know.
